packages/react-percy-config/src/defaults.js

```diff
@@ -1,4 +1,4 @@
 export default {
-  ignoreRegexes: [/node_modules/],
```

Comment:
We previously had an `ignoreRegexes` config option, but it wasn't used anywhere. The new `snapshotIgnorePatterns` config option is actually respected.